components/package.json

```diff
@@ -31,16 +31,16 @@
         "@angular/material": "^8.2.3",
         "@angular/platform-browser": "~8.2.2",
         "@angular/platform-browser-dynamic": "~8.2.2",
-        "@pxblue/eslint-config": "^1.1.4",
+        "@pxblue/eslint-config": "^2.0.1-beta.1",
```

Comment:
Let's avoid using beta packages in the dependency arrays for our packages when we can. If this is ready for a full publish, i'll promote it today.

Reply:
I haven't used this package to test any react code yet, but from an angular perspective it's good to be published.